<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>505</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Vanessmark Desgloses 1.0.2</string>
  </property>
  <property name="windowIcon">
   <iconset resource="vmDesgloses.qrc">
    <normaloff>:/MainWindow/resources/icono_Vanessmark.png</normaloff>:/MainWindow/resources/icono_Vanessmark.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>655</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Desgloses</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="actionBuscar"/>
    <addaction name="actionEditar"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca"/>
   </widget>
   <widget class="QMenu" name="menuConfiguracion">
    <property name="title">
     <string>Base de datos</string>
    </property>
    <addaction name="actionConfigBD"/>
    <addaction name="actionCopiarRestaurar"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfiguracion"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionBarNuevo"/>
   <addaction name="actionBarBuscar"/>
   <addaction name="actionBarEditar"/>
   <addaction name="separator"/>
   <addaction name="actionBarImprimir"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNuevo">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/icono_Nuevo.png</normaloff>:/MainWindow/Resources/icono_Nuevo.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
   <property name="toolTip">
    <string>Inserta nuevos clientes y desgloses</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="actionBuscar">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/icono_Buscar.png</normaloff>:/MainWindow/Resources/icono_Buscar.png</iconset>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </action>
  <action name="actionBarNuevo">
   <property name="icon">
    <iconset resource="vmDesgloses.qrc">
     <normaloff>:/MainWindow/resources/icono_Nuevo.png</normaloff>:/MainWindow/resources/icono_Nuevo.png</iconset>
   </property>
   <property name="text">
    <string>BarNuevo</string>
   </property>
   <property name="toolTip">
    <string>Inserta nuevos clientes y desgloses</string>
   </property>
  </action>
  <action name="actionAcerca">
   <property name="text">
    <string>Acerca</string>
   </property>
  </action>
  <action name="actionBarBuscar">
   <property name="icon">
    <iconset resource="vmDesgloses.qrc">
     <normaloff>:/MainWindow/resources/icono_Buscar.png</normaloff>:/MainWindow/resources/icono_Buscar.png</iconset>
   </property>
   <property name="text">
    <string>BarBuscar</string>
   </property>
   <property name="toolTip">
    <string>Busca desgloses de los clientes</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionBarEditar">
   <property name="icon">
    <iconset resource="vmDesgloses.qrc">
     <normaloff>:/MainWindow/resources/icono_Cliente.png</normaloff>:/MainWindow/resources/icono_Cliente.png</iconset>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
   <property name="toolTip">
    <string>Edita los desgloses</string>
   </property>
  </action>
  <action name="actionBarImprimir">
   <property name="icon">
    <iconset resource="vmDesgloses.qrc">
     <normaloff>:/MainWindow/resources/icono_Impresora.png</normaloff>:/MainWindow/resources/icono_Impresora.png</iconset>
   </property>
   <property name="text">
    <string>Impresora</string>
   </property>
   <property name="toolTip">
    <string>Imprime los desgloses</string>
   </property>
  </action>
  <action name="actionConfigBD">
   <property name="text">
    <string>Datos conexion</string>
   </property>
  </action>
  <action name="actionEditar">
   <property name="icon">
    <iconset>
     <normaloff>:/MainWindow/Resources/icono_Cliente.png</normaloff>:/MainWindow/Resources/icono_Cliente.png</iconset>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
  </action>
  <action name="actionCopiarRestaurar">
   <property name="text">
    <string>Copiar y restaurar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="vmDesgloses.qrc"/>
 </resources>
 <connections/>
</ui>
